### PR TITLE
Patreon.com | Presence

### PR DESCRIPTION
**Pull request by ReXulEc**
Haii, it's me (again 🙌)

I changed the metadata and added new features and pages to the presence section.
I hope you like it!!

https://i.imgur.com/HMMTYMn.jpg ( Thats a part of it, not all :D )

See ya!

### CODE REVIEW

File: websites/P/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Patreon",
+    "author": {
+      "name": "rexulec",
+      "id": "709376748206293002"
+  },
+  "category": "socials",
+  "logo": "https://i.imgur.com/d0YtwcF.png",
```
Comment (Bas950) [inline websites/P/dist/metadata.json:9]: 541x540... really that 1px...

File: websites/P/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "donate",
+    "subscription",
+    "support"
+  ],
+    "url": ["www.patreon.com", "patreon.com"],
+    "description": {
+      "en": "Patreon, is an Internet-based platform that allows creators to set up their own subscription content services.",
+      "tr": "Patreon, içerik oluşturucularına kendi abonelik içeriği hizmetlerini kurmalarına izin veren, İnternet tabanlı bir platformdur.",
+      "de": "Patreon,ist eine internetbasierte Plattform, mit der Entwickler ihre eigenen Abonnementinhaltsdienste einrichten können."  
+    }
```
Comment (Bas950) [inline websites/P/dist/metadata.json:25]:
```suggestion
    "description": {
      "en": "Patreon is a membership platform that makes it easy for artists and creators to get paid. Join over 200,000+ creators earning salaries from over 4 million monthly patrons.",
      "nl": "Patreon is een lidmaatschapsplatform waarmee artiesten en makers gemakkelijk betaald kunnen worden. Sluit je aan bij meer dan 200.000+ videomakers die salarissen verdienen van meer dan 4 miljoen maandelijkse patrons."
    }
```
Use the official description, added it for Dutch, don't speak German or Turkish so can't translate it to those...

File: websites/P/presence.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  presence.setActivity(pagenotfound);
+}
+  else if (document.location.pathname.startsWith("/")) { //
+    const unsupportedPage = {
+        details: "Viewing a",
+        state: "Unsupported page",
+        largeImageKey: "logo"
+  };
+    presence.setActivity(unsupportedPage);
+  }
```
Comment (Bas950) [inline websites/P/presence.ts:165]: Just do 
```
presence.setActivity({
        details: "Viewing a",
        state: "Unsupported page",
        largeImageKey: "logo"
  });
```

Or make an object in the beginning, assign values to that, and then do setActivity once at the end.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
basically - not enough yet for this very "complicated" site :D

**Pull request closed without merging**